src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -42,6 +42,12 @@ public StorageOperationException(String message) {
             super(message);
         }
     }
+    
+    public static class StorageFileNotFoundException extends Exception {
```

Comment:
Good idea to introduce a new exception class. But keep in mind to maintain these separately rather than within the StorageFile is self.